testbed/correctness/util.go

```diff
@@ -0,0 +1,152 @@
+package correctness
```

Comment:
Can this refactoring be split in a separate PR?

Reply:
Sure

Reply:
Separate PR is here: https://github.com/open-telemetry/opentelemetry-collector/pull/1497